src/docfx/build/Build.cs

```diff
@@ -53,13 +53,13 @@ public static async Task Run(string docsetPath, CommandLineOptions options, Erro
             using (var context = new Context(outputPath, errorLog, docset, () => xrefMap))
             {
                 xrefMap = XrefMapBuilder.Build(context, docset);
-                var tocMap = TableOfContentsMap.Create(context, docset);
+                var tocMap = TableOfContentsMap.Create(context);
 
-                context.BuildQueue.Enqueue(GetBuildScope(docset, tocMap));
+                context.BuildQueue.Enqueue(context.BuildScope.Files);
```

Comment:
we are moving more and more things into context, what's the different to keep them inside of docset?

Reply:
I'm thinking eventually there will be no `Docset` class 😁, `Document` becomes a pure data model, and everything else goes to their respective behavioral classes (e.g., resolveAlias goes to `DependencyResolver`)